dali/python/nvidia/dali/ops.py

```diff
@@ -601,29 +601,103 @@ def _choose_device(inputs):
         return "gpu"
     return "cpu"
 
-def _target_uniform_dev(inputs, dev):
-    return inputs
-
-
-def _arithm_op(name, *inputs):
-        input_desc = ""
-        for i, input in enumerate(inputs):
-            input_desc += "&" + str(i)
-            if i < len(inputs) - 1:
-                input_desc += " "
-        expression_desc = "{}({})".format(name, input_desc)
-        dev = _choose_device(inputs)
-        # Create "instance" of operator
-        op = ArithmeticGenericOp(device = _choose_device(inputs), expression_desc = expression_desc)
-        # If we are on gpu, we must mark all inputs as gpu
-        if dev == "gpu":
-            dev_inputs = list(input.gpu() for input in inputs)
+def _is_integer_like(input):
+    if type(input) == int:
+        return True
+    if isinstance(input, Constant):
+        if input.dtype in [DALIDataType.INT8, DALIDataType.INT16, DALIDataType.INT32,
```

Comment:
Can't you use just `_int_types` from types.py?

Reply:
Done.